mysql/db-init/init.sql

```diff
@@ -1,27 +1,22 @@
-CREATE TABLE translations (
-`id` BIGINT(20) UNSIGNED AUTO_INCREMENT PRIMARY KEY,
-`from` VARCHAR(20),
-`source` TEXT,
-`translation` TEXT,
-`created_at` TIMESTAMP(6) NOT NULL DEFAULT CURRENT_TIMESTAMP(6)
-);
-
 CREATE TABLE emails2send (
 `id` BIGINT(20) AUTO_INCREMENT PRIMARY KEY,
-`name` TINYTEXT,
-`phone` TEXT,
-`email` TEXT,
-`content` TEXT,
+`uid` VARCHAR(8) NOT NULL,
+`name` TINYTEXT NOT NULL,
+`phone` TINYTEXT NOT NULL,
```

Comment:
why can't this be null?

Reply:
since we require this in our form (or did I miss smth ?). If phone would be optional, then this could be nullable.